src/store/project.ts

```diff
@@ -325,6 +325,13 @@ export const projectStore = createPartialStore<ProjectStoreTypes>({
             return;
           }
           filePath = ret;
+          if (context.state.projectFilePath) {
+            window.electron.showMessageDialog({
+              type: "info",
+              title: "保存",
+              message: `編集中のプロジェクトが ${filePath} に切り替わりました。`,
+            });
+          }
         }

```

Comment:
すごく細かいのですが、今の使用だと初回保存時じゃない＆overwrite==falseのときにダイアログが出るのですが、初回保存時じゃない＆ファイルパスが切り替わるとき（）にこのダイアログが出るのが直感的に感じました。
こんな感じです。

```suggestion
        }
        if (context.state.projectFilePath && context.state.projectFilePath != filePath) {
          window.electron.showMessageDialog({
            type: "info",
            title: "保存",
            message: `編集中のプロジェクトが ${filePath} に切り替わりました。`,
          });
        }

```

動作は全く一緒だと思うのですが、より直感的かもと思った次第です！

Reply:
修正しました。